NextcloudTalk/SettingsTableViewController.swift

```diff
@@ -0,0 +1,699 @@
+                    let lastUpdate = Date(timeIntervalSince1970: TimeInterval(NCDatabaseManager.sharedInstance().activeAccount().lastContactSync))
+                    let dateFormatter = DateFormatter()
+                    dateFormatter.dateStyle = .medium
+                    dateFormatter.timeStyle = .short
+                    return NSLocalizedString("Last sync: ", comment: "") + dateFormatter.string(from: lastUpdate)
+                }
+            }
+            if settingsSection == SettingsSection.kSettingsSectionUser.rawValue && contactSyncSwitch.isOn {
+                let activeAccount = NCDatabaseManager.sharedInstance().activeAccount()
+                if activeAccount.phone.isEmpty {
+                    let missingPhoneString = NSLocalizedString("Missing phone number information", comment: "")
+                    return "⚠ " + missingPhoneString
+                }
+            }
+        }
+        return nil
+    }
+
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+
+        var cell = UITableViewCell()
+        let videoConfigurationCellIdentifier = "VideoConfigurationCellIdentifier"
+        let browserConfigurationCellIdentifier = "BrowserConfigurationCellIdentifier"
+        let readStatusCellIdentifier = "ReadStatusCellIdentifier"
+        let contactsSyncCellIdentifier = "ContactsSyncCellIdentifier"
+        let privacyCellIdentifier = "PrivacyCellIdentifier"
+        let sourceCodeCellIdentifier = "SourceCodeCellIdentifier"
+        let userStatusCellIdentifier = "UserStatusCellIdentifier"
+
+        let sections = getSettingsSections()
+        let settingsSection = sections[indexPath.section]
+
+        switch settingsSection {
+        case SettingsSection.kSettingsSectionUser.rawValue:
+            let cell = (tableView.dequeueReusableCell(withIdentifier: kUserSettingsCellIdentifier, for: indexPath) as? SettingsUserTableViewCell)!
+            let activeAccount = NCDatabaseManager.sharedInstance().activeAccount()
+            cell.userDisplayNameLabel.text = activeAccount.userDisplayName
+            let accountServer = activeAccount.server
+            cell.serverAddressLabel.text = accountServer.replacingOccurrences(of: "https://", with: "")
+            cell.userImageView.image = NCAPIController.sharedInstance().userProfileImage(for: activeAccount, with: CGSize(width: 160, height: 160))
+            cell.accessoryType = .disclosureIndicator
+            return cell
+        case SettingsSection.kSettingsSectionUserStatus.rawValue:
+            cell = UITableViewCell(style: .default, reuseIdentifier: userStatusCellIdentifier)
+            if activeUserStatus != nil {
+                cell.textLabel?.text = activeUserStatus!.readableUserStatus()
+                let statusImage = activeUserStatus!.userStatusImageName(ofSize: 24)
+                cell.imageView?.image = UIImage(named: statusImage)
```

Comment:
This part is not showing the user status message (if there is any) as in the objective C view controller.